genesyscloud/architect_user_prompt/resource_genesyscloud_architect_user_prompt.go

```diff
@@ -25,14 +26,17 @@ func getAllUserPrompts(ctx context.Context, clientConfig *platformclientv2.Confi
 	resources := make(resourceExporter.ResourceIDMetaMap)
 	proxy := getArchitectUserPromptProxy(clientConfig)
 
-	userPrompts, resp, err := proxy.getAllArchitectUserPrompts(ctx, true, true, "")
-	if err != nil {
-		return nil, util.BuildAPIDiagnosticError(resourceName, fmt.Sprintf("failed to get user prompts: %s", err), resp)
-	}
-	for _, userPrompt := range *userPrompts {
-		resources[*userPrompt.Id] = &resourceExporter.ResourceMeta{Name: *userPrompt.Name}
-	}
+	exportNameFilter := "abcdefghijklmnopqrstuvwxyz"
```

Comment:
there could be names starting with special chars or nums ? also check if we can leverage "name" argument instead of a new argument "exportNames"?